05_speed_test_comparison_dplyr_duckdb.qmd

```diff
@@ -0,0 +1,299 @@
+---
+title: "DuckDB Evaluation"
+date: today
+format: html
+toc: true
+params:
+  n_trials: 5
+  run_loop: no
+editor_options: 
+  chunk_output_type: console
+---
+
+# Introductions
+
+The purpose of this project is to provide a high-level evaluation of [DuckDB](https://duckdb.org) as a solution for projects involving databases at BC Stats.
+
+# Advantages of DuckDB
+
+Section left blank for Jon.
```

Comment:
I think that these sections, which are essentially report summaries, can be left out - use git for hosting the code, and use the LAN for hosting the final report documentation.

Reply:
I should add something important, though -- I recently re-did a lot of the code (using targets for one!) but it's not quite done yet so I haven't pushed it to Github yet. So maybe hold off on the code review until it's up there?? The idea is more or less the same as before and certainly the documentation part won't change.